docs/changelog/1_0_a3.rst

```diff
@@ -0,0 +1,158 @@
+===========
+1.0 Alpha 3
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 3.
+
+
+Rust
+====
+
+We've started the process of rewriting some of the
+performance-critical EdgeDB internals to Rust. So far we have
+converted the lexer as well as the GraphQL parser.
+
+Additionally, we're also making EdgeDB command-line tools using Rust.
+Notably the REPL has been converted and now features more commands:
```

Comment:
All tools have been converted. The goal of the conversion is ease of distribution. The tools are a single binary now and can run on multiple different platforms, including Windows.

Reply:
I'll add the comment about distribution, etc.

Reply:
"Notably the REPL has been converted and now features more commands:" -- this sentence is not needed anymore?

Victor, please do a complete read of this file and fix it.

Reply:
Reworded.